### DISCUSSION TITLE
Product Variation manual sorting

### DISCUSSION THREAD

**Discussion by dcbndev** [lunarphp/lunar#1039] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
While running a clothing shop, every product often has many variations in terms of Colors and Sizes. Sometimes it is required to sort them in a specific order, which at the moment is not possible. Variations could be saved in a specific order when created, but this sorting order can't be modified later on.

**Describe the solution you'd like**
I saw that product options have sorting functionallity with drag & drop, as seen in the screenshot below.
Maybe this kind of drag & drop sorting could be implemented for Product Variations as well, in the variations table?
<img width="676" alt="Screen Shot 2023-03-31 at 16 40 11" src="https://user-images.githubusercontent.com/52117728/229164774-379bebd7-a66d-4ede-afa7-185f6ee5f10d.png">

**Additional context**
The realisation of product options drag & drop sorting.
<img width="742" alt="Screen Shot 2023-03-31 at 18 26 55" src="https://user-images.githubusercontent.com/52117728/229164524-cf44fde5-aca5-4eb6-bf2c-e38bbdd7722d.png">

**Comment by ryanmitchell** [comment]
We would also love to see the ability to sort variants to have finer control over the order they can be output on the front end side.
